apollo-server/v4/generated-types/package.json

```
@@ -0,0 +1,30 @@
+{
+  "name": "as4-generating-types-example",
+  "version": "1.0.0",
+  "description": "An example of generating types from a schema to use in your resolvers.",
+  "main": "index.js",
+  "type": "module",
+  "scripts": {
+    "generate": "graphql-codegen --config codegen.yml",
+    "compile": "npm run generate && tsc",
+    "postinstall": "npm run compile",
+    "start": "npm run compile && node ./dist/index.js",
+    "watch": "concurrently  --names \"CodeGen,TS,Nodemon\" -c \"bgYellow,bgBlue,bgMagenta\" \"graphql-codegen --watch 'src/**/*.ts'\"  \"tsc -w \" \"nodemon ./dist/index.js \""
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@apollo/server": "^4.0.0-alpha.10",
+    "graphql": "^16.6.0",
+    "graphql-tag": "^2.12.6",
+    "typescript": "^4.8.3"
+  },
+  "devDependencies": {
+    "@graphql-codegen/cli": "2.12.0",
+    "@graphql-codegen/typescript": "2.7.3",
+    "@graphql-codegen/typescript-resolvers": "2.7.3",
+    "concurrently": "^7.4.0",
+    "nodemon": "^2.0.19"
```

Not a huge deal here, but for consistency. Our tendency is to pin dev dependencies. This change will need to be accompanied by an `npm i` in order to update the package-lock as well.

```suggestion
    "concurrently": "7.4.0",
    "nodemon": "2.0.19"
```